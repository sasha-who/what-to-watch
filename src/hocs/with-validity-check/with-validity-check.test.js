import * as React from "react";
import renderer from "react-test-renderer";
import withValidityCheck from "./with-validity-check.js";

const MockComponent = () => {
  return (
    <div></div>
  );
};

const MockComponentWrapped = withValidityCheck(MockComponent);

it(`withValidityCheck should render correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
