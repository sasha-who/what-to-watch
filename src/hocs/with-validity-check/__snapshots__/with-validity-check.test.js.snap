// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withValidityCheck should render correctly 1`] = `<div />`;
